api/src/unraid-api/graph/resolvers/backup/source/backup-source.types.ts

```
@@ -0,0 +1,136 @@
+    scriptConfig?: ScriptPreprocessConfig;
+
+    @Field(() => RawBackupConfig, { nullable: true })
+    rawConfig?: RawBackupConfig;
+}
+
+export const SourceConfigUnion = createUnionType({
+    name: 'SourceConfigUnion',
+    types: () =>
+        [ZfsPreprocessConfig, FlashPreprocessConfig, ScriptPreprocessConfig, RawBackupConfig] as const,
+    resolveType(obj: any, context, info) {
+        if (ZfsPreprocessConfig.isTypeOf && ZfsPreprocessConfig.isTypeOf(obj)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)